x-pack/build_chromium/build.py

```diff
@@ -136,3 +145,6 @@ def archive_file(name):
 print('Creating ' + path.join(src_path, md5_filename))
 with open (md5_filename, 'w') as f:
   f.write(md5_file(zip_filename))
+
+runcmd('gsutil cp ' + path.join(src_path, zip_filename) + ' gs://headless_shell_staging')
+runcmd('gsutil cp ' + path.join(src_path, md5_filename) + ' gs://headless_shell_staging')
```

Comment:
cc @spalger : this adds an auto-upload to a staging bucket as we talked about. 

I think this suffices the problem we had here: https://github.com/elastic/kibana/pull/98688#issuecomment-828856086

This is without adding a dry-run option to the script, as discussed.  LMK if you think it needs the dry-run option. (It would be a pain since a run takes hours.)

Reply:
Yeah, I think this is the best approach, then staging assets can just be `gsutil mv`d to the `headless_shell` bucket when they're verified.

Might be nice to use an environment variable to switch out the URLs to actually download from staging so we can test things before promoting them.